test/docfx.Test/template/JavascriptEngineTest.cs

```diff
@@ -19,6 +19,7 @@ public void RunJavascript(string input, string output)
         {
             var inputJson = JObject.Parse(input.Replace('\'', '"'));
             var outputJson = _js.Run("index.js", "main", inputJson);
+            (outputJson as JObject).Property("__global")?.Remove();
```

Comment:
we do not need `__global` in the output model anymore?

Reply:
yep, this is a temp variable used inside JINT